### PR TITLE
Default to PostgreSQL 9.4

### PR DESCRIPTION
**Pull request by kaapstorm**
@snopoke buddies @sravfeyn @calellowitz

### CODE REVIEW

File: ansible/roles/postgresql/defaults/main.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 pg_install: True
-postgresql_version: '9.1'
+postgresql_version: '9.4'
```
Comment (sravfeyn) [inline ansible/roles/postgresql/defaults/main.yml]: Does this place also need a version change? https://github.com/dimagi/commcarehq-ansible/blob/pg94/ansible/roles/common_installs/defaults/main.yml#L1

Comment (kaapstorm) [inline ansible/roles/postgresql/defaults/main.yml]: Yes! Thank you! I was sure there were two places, but I only found this one. I'll update.

**Comment by kaapstorm** [comment]
:white_check_mark:

**Comment by snopoke** [comment]
How come it's defined in 2 places? Can't we change that?

**Comment by kaapstorm** [comment]
That's a good idea, @snopoke. The one place is for the client and the other for the server, but iirc the client is common to all hosts, and the server should just use the client's version. It's on my to-do list.

**Pull request merged**